src/Files.App/Services/FileTagsService.cs

```diff
@@ -42,7 +42,7 @@ public async IAsyncEnumerable<TaggedItemModel> GetItemsForTagAsync(string tagUid
 		{
 			foreach (var item in FileTagsHelper.GetDbInstance().GetAll())
 			{
-				if (!item.Tags.Contains(tagUid))
+				if (!item.Tags.Contains(tagUid) || RecycleBinHelpers.IsPathUnderRecycleBin(item.FilePath))
```

Comment:
Is it worth splitting this onto multiple lines?

Reply:
I think it's good as it is, but still we should consider adding a `line length limit` to the style guidelines